libindy/src/api/mod.rs

```diff
@@ -116,6 +116,18 @@ pub enum ErrorCode
     // Returned if provided wallet query is invalid
     WalletQueryError = 214,
 
+    // Returned if wallet input has invalid structure (currently used in wallet import)
+    WalletStructureError = 215,
+
+    // Returned if the wallet is not empty, but it is required (currently only when importing)
+    WalletNotEmpty = 216,
+
+    // Returned if path provided to wallet export already exists
+    WalletExportPathExists = 217,
```

Comment:
I suggest to just raise CommonIOError here

Reply:
I wouldnt say its IO error. It is more of an configuration error